ddht/v5_1/alexandria/partials/proof.py

```diff
@@ -312,6 +273,17 @@ def get_tree(self) -> ProofTree:
         return self._tree_cache
 
     def serialize(self) -> bytes:
+        """
+        Proof serialization is the concatenation of:
+
+        - the LEB128 encoded `content_length`
```

Comment:
The refactor and comments are a nice gain in readability 🚀 nit: would be nice to know the expected byte-count or range for each of these items, e.g., `- the LEB128 encoded 'content_length' (X bytes)`, etc.